drivers/sip_svc/Kconfig

```diff
@@ -0,0 +1,92 @@
+# Arm SiP services driver configuration options
+
+# Copyright (c) 2021, Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ARM_SIP_SVC
+	bool "Arm SiP services drivers"
+	help
+	  Enable Arm SiP services drivers configuration
```

Comment:
I'm very confused about this. Is `ARM_SIP_SVC` supposed to be a device driver class? As in: do we expect to have several different implementations? Or this is the one single implementation?

Reply:
There is only one implementation for the driver, there can be 2 driver instances, 1 for SMC, 1 for HVC.